core/tests/protractor_utils/admin.js

```diff
@@ -59,6 +59,21 @@ var editConfigProperty = function(
   );
 };
 
+var updateRole = function(name, newRole) {
+  general.waitForSystem();
+  browser.get(general.ADMIN_URL_SUFFIX);
+  element(by.css('.protractor-test-admin-roles-tab')).click();
+  browser.waitForAngular();
+  //change values for update form and submit it.
```

Comment:
// Change values for the "update roles" form, and submit it.

(Note the space after "//", starting with a capital letter, attempt to avoid ambiguity with the verb "update", etc. Write in a way that makes it easy for people to read.)